examples/gno.land/p/demo/avl/pager/pager.gno

```diff
@@ -36,12 +37,13 @@ type Item struct {
 }
 
 // NewPager creates a new Pager with default values.
-func NewPager(tree *avl.Tree, defaultPageSize int) *Pager {
+func NewPager(tree *avl.Tree, defaultPageSize int, reversed bool) *Pager {
```

Comment:
nit: maybe have `NewReversePager` instead

Reply:
I think this API is more concise, and this feature is more easily discoverable this way, since you have to specify the bool.